apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -884,41 +885,50 @@ const Main = ({
     rawData.eventTypeGroups.length === 1;
 
   const data = denormalizePayload(rawData);
-
   return (
     <>
       {data.eventTypeGroups.length > 1 || isFilteredByOnlyOneItem ? (
         <>
           {isMobile ? (
             <MobileTeamsTab eventTypeGroups={data.eventTypeGroups} />
           ) : (
-            data.eventTypeGroups.map((group, index: number) => (
-              <div
-                className="mt-4 flex flex-col"
-                data-testid={`slug-${group.profile.slug}`}
-                key={group.profile.slug}>
-                <EventTypeListHeading
-                  profile={group.profile}
-                  membershipCount={group.metadata.membershipCount}
-                  teamId={group.teamId}
-                  bookerUrl={group.bookerUrl}
-                />
+            data.eventTypeGroups.map((group, index: number) => {
+              const userHasManagedEventType = group.eventTypes.find(
+                (event) => event.metadata?.managedEventConfig
+              );
+              if (group.profile.eventTypesLockedByOrg && !userHasManagedEventType) return null;
```

Comment:
> - what about the hidden event types? shouldn't we still show them
> - we should also lock the toggle of the hidden event types

I don't see any changes here, these two things are still an issue for me. 

Example: Admin locks event types for members. Member has one personal event type (no managed event type). Then we want to mark the event type as hidden and it should still show in the event type list. Right now the hidden event types are not shown if the user doesn't have a managed event type. Also the toggle to make the event type public again should be disabled for the member

Reply:
I think I forgot to push some code 😅

Reply:
Commit history looks weird as the feedback was timestamped before this message :P 

Latest changes should be there